services/backend/src/core/search/util/getAllProfiles.js

```diff
@@ -119,6 +119,20 @@ async function getProf(profile, searchValue) {
     return null;
   });
 
+  const mentorshipSkills = await profile
+    .getMentorshipSkills()
+    .map(async (skill) => {
+      if (skill)
+        return {
+          id: skill.dataValues.id,
+          description: {
+            en: skill.dataValues.descriptionEn,
+            fr: skill.dataValues.descriptionFr,
+          },
+        };
+      return null;
+    });
+
```

Comment:
Surprisingly, I think only this part needs to updated because of the new backend (aside from the merge conflicts)